zulip_bots/zulip_bots/bots/incrementor/incrementor.py

```diff
@@ -1,31 +1,33 @@
 # See readme.md for instructions on running this code.
 
+from zulip_bots.lib import ExternalBotHandler
 
 class IncrementorHandler(object):
     META = {
         'name': 'Incrementor',
         'description': 'Example bot to test the update_message() function.',
     }
 
-    def usage(self):
+    def usage(self) -> str:
         return '''
         This is a boilerplate bot that makes use of the
         update_message function. For the first @-mention, it initially
         replies with one message containing a `1`. Every time the bot
         is @-mentioned, this number will be incremented in the same message.
         '''
 
-    def initialize(self, bot_handler):
+    def initialize(self, bot_handler: ExternalBotHandler) -> None:
         storage = bot_handler.storage
         if not storage.contains('number') or not storage.contains('message_id'):
-            storage.put('number', 0)
-            storage.put('message_id', None)
+            # `storage.put` only accepts `str`s.
+            storage.put('number', str(0))
+            storage.put('message_id', '')
 
-    def handle_message(self, message, bot_handler):
+    def handle_message(self, message: dict, bot_handler: ExternalBotHandler) -> None:
```

Comment:
This should be `Dict`. You'll also need to specify what it is a `Dict` of, ie what other types.
These are elsewhere too.